packages/compass-aggregations/test/configure-store.ts

```diff
@@ -86,7 +87,9 @@ export default function configureStore(
 export function renderWithStore(
   ui: React.ReactElement,
   ...args: Parameters<typeof configureStore>
-) {
+): Promise<
+  RenderPluginWithConnectionsResult<typeof CompassAggregationsPlugin.provider>
```

Comment:
... and sometimes I only managed to make it work by explicitly re-annotating fuctions as the error text suggests doing